src/nav/new-relic-solutions.yml

```diff
@@ -149,7 +149,21 @@ pages:
   - title: Build on New Relic UI
     pages:
       - title: New Relic CLI
-        path: /docs/new-relic-solutions/tutorials/new-relic-cli
+        path: /docs/new-relic-solutions/tutorials/new-relic-cli    
+      - title: New Relic One CLI commands
+        pages: 
+          - title: Common commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-common
+          - title: Config commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-config
+          - title: Nerdpack commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-nerdpack
+          - title: Subscription commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-subscription
+          - title: Plugin commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-plugins
+          - title: Catalog commands
+            path: /docs/new-relic-solutions/build-nr-ui/nr1-cli/nr1-catalog
       - title: Build your own New Relic apps
         path: /docs/new-relic-solutions/new-relic-one/build-custom-new-relic-one-application
       - title: Create a "Hello, World!" app in New Relic
```

Comment:
The quotes in this line were causing the issue

Reply:
Thank you so much, @clarkmcadoo. It's working now!